src/Term_Command.php

```diff
@@ -235,8 +235,16 @@ public function create( $args, $assoc_args ) {
 	 * <taxonomy>
 	 * : Taxonomy of the term to get
 	 *
-	 * <term-id>
-	 * : ID of the term to get
+	 * <term>
+	 * : ID or slug of the term to get
+	 *
+	 * [--by=<field>]
+	 * : Explicitly handle the term value as a slug or id.
+	 * ---
+	 * options:
```

Comment:
Include `default: id` in this argument definition to have a default value always passed through.